dialogy/types/intent/__init__.py

```diff
@@ -246,5 +243,5 @@ def json(self) -> Dict[str, Any]:
             "alternative_index": self.alternative_index,
             "score": self.score,
             "parsers": self.parsers,
-            "slots": [slot.json() for slot in self.slots.values()],
-        }
+            "slots": [slot.dict() for slot in self.slots.values()],
```

Comment:
What's the difference between `.json()` and `.dict()`? As far as I can see pydantic supports both..

Reply:
Understood... json will give a `str` representation and `dict` will give a python dict representation